ipaserver/install/ipa_replica_prepare.py

```diff
@@ -427,7 +427,7 @@ def copy_httpd_certificate(self):
             self.log.info("Creating SSL certificate for the Web Server")
             self.export_certdb("httpcert", passwd_fname)
 
-            self.log.info("Exporting RA certificate")
+            self.log.info("Copying RA certificate")
```

Comment:
Nitpick: IMO the original message is better, copying is an implementation detail of the exporting.